### PR TITLE
Add support for go-gettable tools

### PR DESCRIPTION
**Pull request by biochimia**

### CODE REVIEW

File: get-git/getter.go [hunk 1]
```diff
@@ -20,32 +21,41 @@ import (
...
+	Branch Branch
+}
 
+type Getter struct {
+	Options
 	gitDir string
 }
 
-func (gg gitGetter) runNativeGit(command string, args ...string) error {
+func (gg Getter) runNativeGit(command string, args ...string) error {
```
Comment (paulbes) [inline get-git/getter.go]: Not really a thing, but this double `gg` is killing my OCD

**Review by paulbes — APPROVED** [review]
We are kind of replicating the behaviour of

File: get-git/getter.go (hunk 1 above)
Comment (biochimia) [reply get-git/getter.go]: You want a single one?

Comment (biochimia) [reply get-git/getter.go]: I had noticed, but was trying to ignore it.

**Pull request merged**